src/core.js

```diff
@@ -147,6 +147,13 @@ extend( QUnit, {
 	// DEPRECATED: QUnit.stop() will be removed in QUnit 2.0.
 	stop: function( count ) {
 
+		var warning;
+
+		if ( this === global ) {
+			warning = QUnit.warning.deprecated.stop;
+			QUnit.warning(warning);
```

Comment:
We may avoid the new `warning` variable calling the function directly with it, less use of memory.

Also: I still believe it's better to map the messages as string keys, the point is that we may want to trigger only once for each warning. If you find another way to solve this (the single trigger) you don't need to worry on the key map.